DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: toolchest
 Title: R Client for Toolchest
-Version: 0.5.0
+Version: 0.7.0
```

Comment:
Are you thinking we match minor but not patch numbers Python <-> R?

Reply:
That's what I was thinking; if we need to update something on one side but not the other, things could get out of hand.